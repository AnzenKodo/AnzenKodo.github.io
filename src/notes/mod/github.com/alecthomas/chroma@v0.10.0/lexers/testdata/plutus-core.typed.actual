(program 1.0.0
    [
        (lam f (all b (type) (fun b b)) [{f (con integer)} (con integer 5)])
        (abs a (type) (lam x a x))
    ]
)
