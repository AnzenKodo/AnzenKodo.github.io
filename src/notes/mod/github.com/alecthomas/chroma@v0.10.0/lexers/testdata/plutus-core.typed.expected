[
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"program "},
  {"type":"LiteralString","value":"1.0.0"},
  {"type":"Text","value":"\n    "},
  {"type":"Punctuation","value":"["},
  {"type":"Text","value":"\n        "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"lam "},
  {"type":"Name","value":"f"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"all "},
  {"type":"Name","value":"b"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"type"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"fun "},
  {"type":"Name","value":"b"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"b"},
  {"type":"Punctuation","value":"))"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"[{"},
  {"type":"Name","value":"f"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"con "},
  {"type":"KeywordType","value":"integer"},
  {"type":"Punctuation","value":")}"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"con "},
  {"type":"KeywordType","value":"integer"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"5"},
  {"type":"Punctuation","value":")])"},
  {"type":"Text","value":"\n        "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"abs "},
  {"type":"Name","value":"a"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"type"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Keyword","value":"lam "},
  {"type":"Name","value":"x"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"a"},
  {"type":"Text","value":" "},
  {"type":"Name","value":"x"},
  {"type":"Punctuation","value":"))"},
  {"type":"Text","value":"\n    "},
  {"type":"Punctuation","value":"]"},
  {"type":"Text","value":"\n"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":"\n"}
]
